CMSgov/mint-app
# [EASI-2055] Front End for Beneficiaries

# EASI-2055

## Changes and Description

- Figma link: https://www.figma.com/file/M72DReE73SSXMN48WZfaKN/MINT---New-Model-Process-(Model-Intake)?node-id=1206%3A88200 
- Add three pages of Beneficiaries to the task list



## How to test this change

1. Navigate to Task List of a model plan
2. Find that beneficiaries is "Start"-able
3. Ensure all pages match the figma


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
